### DISCUSSION TITLE
无法连接E5 OD账户

### DISCUSSION THREAD

**Discussion by Muska-Ami** [spencerwooo/onedrive-vercel-index#950] [Q&A] [answered]
使用的为本人E5管理员账户,之前用一直好好的,现在连不上,已尝试删除upstash数据，从头重新部署，依然无法使用
fork: [https://github.com/XiaMoHuaHuo-CN/onedrive-vercel-index/tree/main](https://github.com/XiaMoHuaHuo-CN/onedrive-vercel-index/tree/main)

已确认upstash正常:

![image](https://user-images.githubusercontent.com/79651409/230751723-c85ad33f-5bea-4a65-ad51-d71850b7b18a.png)

Vercel log:

![image](https://user-images.githubusercontent.com/79651409/230751753-15ee5f30-d249-4096-82d1-0e2415574729.png)

网盘:

![image](https://user-images.githubusercontent.com/79651409/230751782-c0d2fb3c-a101-458e-b251-e1d52108950c.png)

API返回:

```json
{"error":{"error":{"code":"itemNotFound","message":"The resource could not be found.","innerError":{"date":"2023-04-09T02:57:58","request-id":"774440e7-ee18-4e87-87c8-07b78da162a8","client-request-id":"774440e7-ee18-4e87-87c8-07b78da162a8"}}}}
```

> 我盘里明明有文件啊喂
> ![image](https://user-images.githubusercontent.com/79651409/230751824-b8036cb6-8079-4fdf-bcfc-b94418e5bbd5.png)

**Comment by spencerwooo** [comment, marked as answer]
需要按照你自己的 OneDrive 分享路径修改[你 fork 仓库里的配置文件](https://github.com/XiaMoHuaHuo-CN/onedrive-vercel-index/blob/7a97b3fc7eca9757ce6118d96505f7f9c6a15422/config/site.config.js#L22-L23
)：

```diff
  // The folder that you are to share publicly with onedrive-vercel-index. Use '/' if you want to share your root folder.
- baseDirectory: '/Public',
+ baseDirectory: '/',
```
